grpcserver/inputhandler.go

```diff
@@ -47,7 +47,10 @@ func (srv *InputHandler) Start(grpcTLS credentials.TransportCredentials, endpoin
 	var err error
 
 	srv.log.Debugf("initializing the gRPC server...")
-	opts := []grpc.ServerOption{grpc.Creds(grpcTLS)}
+	opts := []grpc.ServerOption{
+		grpc.Creds(grpcTLS),
+		grpc.MaxRecvMsgSize(64 * (1 << 20)), // Allow RPCs of up to 64MB
```

Comment:
Nit: Looks like `elastic-agent-shipper` already indirectly pulls in the https://pkg.go.dev/github.com/docker/go-units#pkg-constants package.  So you could use `64 * MiB` here if you wanted for self-documenting code.